geonode/layers/views.py

```diff
@@ -245,11 +246,21 @@ def layer_detail(request, layername, template='layers/layer_detail.html'):
 
     if layer.storeType == "remoteStore":
         service = layer.service
+        source_url = service.base_url
+        components = urlparse.urlsplit(service.base_url)
+        query_params = None
+        if components.query:
+            query_params = dict(urlparse.parse_qsl(components.query))
```

Comment:
@travislbrundage, does `parse_qsl` need `keep_blank_values=True` param? I mean, are empty `key=` query pairs important here?

Otherwise, everything else in this PR looks good to me.

Reply:
I suppose if they are there, I should probably elect to keep them, although I don't think it's correct if they are empty. So not sure if that is a possibility, but if it is, this probably isn't the code that should make the judgment. Maybe there is a situation where a blank query value is wanted, so I'll make that adjustment. Thanks for the review!